niimprint/niimbot_web_api.py

```diff
@@ -0,0 +1,15 @@
+import requests
+
+
+BASE_URL = 'https://print.niimbot.com/api'
+GET_CLOUD_TEMPLATE_BY_ONE_CODE_URL = BASE_URL + '/template/getCloudTemplateByOneCode'
+
+HEADERS = {
+    'niimbot-user-agent': 'AppId/com.gengcon.jccloudprinter OS/Android AppVersionName/4.9.3 Model/Samsung S20 SystemVersion/10.0 DeviceId/456789 referer/CP001Mobile',
+    'Content-Type': 'application/json',
+}
```

Comment:
What about reducing the header? In my test, putting just `{'niimbot-user-agent': 'AppVersionName/1'}` as a header worked well.

Reply:
It's long because I want to mimic the header of the official app, so in the future the request will still work even if they decide to validate `niimbot-user-agent` header.